### DISCUSSION TITLE
How to get the route path in custom server?

### DISCUSSION THREAD

**Discussion by weyert** [vercel/next.js#23591] [Help]
I am wondering if it's possible to get the `route path` when using a customer server and the request handler function? I am happy if it would be available after running it. I am using the following:

```
app.all('/*', async (req: express.Request, res: express.Response) => {
          try {
            await nextRequestHandler(req, res)
          } finally {
              // NOOP
          }
        })
```

**Comment by jamesmosier** [comment]
What `route path` are you looking for? You can get the route params via `req.params` and the query via `req.query`. Is there something else you are after though?

**Comment by ats1999** [comment]
Can  anyone tell me how to resolve this issue https://github.com/vercel/next.js/discussions/23476

I haven't got any response yet.

Everything is working fine in deployment but not working in development on the local machine.
